python/dgl/graphbolt/data_block.py

```diff
@@ -19,25 +19,23 @@ class DataBlock:
     consistency and ease of use throughout the loading process."""
 
     sampled_subgraphs: List[SampledSubgraph] = None
-    """
-    A list of 'SampledSubgraph's, each one corresponding to one layer,
+    """A list of 'SampledSubgraph's, each one corresponding to one layer,
     representing a subset of a larger graph structure.
     """
 
-    node_feature: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
-    """A representation of node feature.
-    - If `node_feature` is a tensor: It indicates the graph is homogeneous.
-    - If `node_feature` is a dictionary: The keys should be node type and the
-      value should be corresponding node feature or embedding.
+    node_feature: Dict[Tuple[str, str], torch.Tensor] = None
+    """A representation of node features.
+    Keys are tuples of '(node_type, feature_name)' and the values are
+    corresponding features. Note that for a homogeneous graph, where there are
+    no node types, 'node_type' should be None.
```

Comment:
what about using str for homo and Tuple[str, str] for hetero?

Reply:
It will make the code more complicted and inconsistent with feature store.